### ISSUE TITLE
Error when compiling on mac

### ISSUE THREAD

**Issue by blockinator** [PIVX-Project/PIVX#1005]
When compiling PIVX on MacOS Mojave I get the following error when running 'make':

```
qt/pivx/pwidget.cpp:7:10: fatal error: 'qt/pivx/moc_pwidget.cpp' file not found
#include "qt/pivx/moc_pwidget.cpp"
```

I followed this guide to install dependencies and build:
https://github.com/PIVX-Project/PIVX/blob/master/doc/build-osx.md

**Comment by akshaynexus** [comment]
are you compiling from the master?if so try doing bash autogen.sh and then try make.If not then list what was the steps taken to get that error.

**Comment by blockinator** [comment]
I am compiling from master. I ran the 'bash autogen.sh' and 'make' and it fixed the ```qt/pivx/moc_pwidget.cpp' file not found``` error. But now I am getting this error:

```
qt/pivx/furabstractlistitemdelegate.cpp:6:10: fatal error: 
      'qt/pivx/moc_furabstractlistitemdelegate.cpp' file not found
#include "qt/pivx/moc_furabstractlistitemdelegate.cpp"
```

**Comment by akshaynexus** [comment]
try ```make clean && bash autogen.sh && make```

**Comment by blockinator** [comment]
I tried the 'make clean && bash autogen.sh && make' and it didn't work so I recloned the repository and ran 'bash autogen.sh && ./configure && make' and got the original error.

```
qt/pivx/pwidget.cpp:7:10: fatal error: 'qt/pivx/moc_pwidget.cpp' file not found
#include "qt/pivx/moc_pwidget.cpp"
```

**Comment by Fuzzbawls** [comment]
this should be resolved now that #1010 has been merged
